services/client/util.go

```diff
@@ -194,3 +211,22 @@ func (su shardByIDAscending) Less(i, j int) bool {
 func (su shardByIDAscending) Swap(i, j int) {
 	su[i], su[j] = su[j], su[i]
 }
+
+func adKey(sid services.ServiceID, id string) string {
+	return fmt.Sprintf("[%s][%s]", serviceKey(sid), id)
```

Comment:
mostly to try to keep in sync with the old key format in statsdex, but actually "%s/%s" looks simpler and more clean now, unified to that format